src/BtLineEdit.cpp

```diff
@@ -26,8 +26,9 @@
 #include "Algorithms.h"
 #include <QSettings>
 #include <QDebug>
+#include <QStyle>
 
-BtLineEdit::BtLineEdit(QWidget *parent, Unit::UnitType type) :
+BtLineEdit::BtLineEdit(QWidget *parent, Unit::UnitType type, char const * const maximalDisplayString) :
```

Comment:
If we need to internationalize this parameter (for example, something on the Style editor), will this char const*const work? Should we maybe consider a QString?

Reply:
Ah, yes, good point.  Will fix.